### PR TITLE
Fixed some issues regarding permissions in docker and openshift objects

### PR DESCRIPTION
**Pull request by ghost**
Hi.

I have fixed some issues regarding installation with root privileges in Dockerfile.rhel7, a selector problem in deploymentconfig and set up the user in container.

Please, check if this is correct and merge accordingly.

Thanks.

### CODE REVIEW

File: OpenShift/nexus3.json [hunk 1]
```diff
@@ -1,276 +0,0 @@
-{
```
Comment (punkisdead) [inline OpenShift/nexus3.json]: Why did you remove this?

File: Dockerfile.rhel7 [hunk 2]
```diff
@@ -60,8 +60,15 @@ ENV NEXUS_DATA=/nexus-data \
...
 
-RUN mkdir -p ${NEXUS_HOME} && \
-    curl --fail --silent --location --retry 3 \
+RUN useradd -l -u ${USER_UID} -r -g 0 -m -d ${NEXUS_DATA} -s /sbin/no-login \
+            -c "${USER_NAME} application user" ${USER_NAME} && \
+    mkdir -p ${NEXUS_HOME} && \
+    chown ${USER_NAME} ${NEXUS_HOME}
+
+# Supply non variable to USER command ${USER_NAME}
+USER ${USER_NAME}
```
Comment (punkisdead) [inline Dockerfile.rhel7]: Unfortunately passing a variable to the `USER` command fails the Docker linter, which ultimately fails the TravisCI job. Not sure if we can adjust the linter to ignore this or not.

Is there a reason you're using a variable for this instead of just using the `nexus` user like earlier?

File: OpenShift/nexus3.json (hunk 1 above)
Comment (ghost) [reply OpenShift/nexus3.json]: simply move the file to nexus3-rhel.json, so a nexus3-centos.json can also be implemented for openshift origin.

File: Dockerfile.rhel7 (hunk 2 above)
Comment (ghost) [reply Dockerfile.rhel7]: Only reason was trying to keep just one value for being consistent. If Docker linter does not support it, it is ok to use the previous hardcoded value.

File: OpenShift/nexus3.json (hunk 1 above)
Comment (punkisdead) [reply OpenShift/nexus3.json]: Ok, makes sense.
